src/pages/LoginPage.tsx

```diff
@@ -65,12 +51,40 @@ const LoginPage = () => {
   };
 
   return (
-    <div className='field'>
+    <div className="field">
+      {loading &&
+        <h1
+          style={{
+            position: "fixed", left: "50%", transform: "translate(-50%, -40%)", color: "grey"
```

Comment:
please, move it to the scss class (not use inline style except extra situations)

Reply:
this Loading for example, it change when Daniil make new Loading